pydantic/types.py

```diff
@@ -606,3 +607,68 @@ def _get_brand(card_number: str) -> PaymentCardBrand:
         else:
             brand = PaymentCardBrand.other
         return brand
+
+
+BYTE_SIZES = {
+    '': 1,
+    'B': 1,
+    'kB': 10 ** 3,
```

Comment:
about about if someone enters `10k`, that's what `ls` used by default.

Perhaps the `B` is superfluous in the `kB...EB` cases since it's already called `ByteSize`?

Reply:
Added.